content/general/concepts/octal/octal.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: 'Octal' 
+Description: 'Octal is a number system that uses digits from 0 to 7. Compared to the decimal system each digit corresponds to the power of 8 instead of the power of 10.' 
+Subjects: 
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Code Foundations'
+Tags: 
+  - 'Arithmetic'
+  - 'Encoding'
+CatalogContent: 
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+---
+
+The **octal** number system can be compared with the decimal, the hexadecimal, the [binary](https://www.codecademy.com/resources/docs/general/binary), and many other systems. As the name says the system consists of 8 different digits (0-7). In our daily life, we are used to the decimal system, which consists of steps to the power of 10. Instead the Octal system uses steps to the power of 8.
+
+In the past, the octal system was used more often because systems were based on 12/24/36-bit but in today's technologies, especially with 16/32/64-bit the binary or hexadecimal system is used commonly. In the case of the hexadecimal system, the octal system has the advantage of using only numbers, therefore, renouncing using the alphabet.
```

Comment:
```suggestion
The octal system was often used in the past because systems were based on 12, 24, or 36 bits. The binary or hexadecimal system is commonly used in today's technologies, especially with 16, 32, or 64-bit machines. The octal number system has an advantage over the hexadecimal system in that we do not need to use characters from the alphabet to express a number in octal.
```